data/VISHAL3545.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Vishal Kumar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Data Engineer turned *Content Creator* ",
+  "avatar": "https://github.com/VISHAL3545.png"
+}
+
+{
+  "name": "Vishal Kumar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Data Engineer turned *Content Creator*",
+  "avatar": "https://github.com/SaraJaoude.png",
```

Comment:
you can add your github profile image in line 14 - https://github.com/VISHAL3545.png